### PR TITLE
Define import path in preparation of repository move to jaegertracing

### PR DESCRIPTION
**Pull request by yurishkuro**
per https://github.com/uber/jaeger/issues/409

### CODE REVIEW

File: crossdock/services/tracehandler.go [hunk 1]
```diff
@@ -129,10 +129,7 @@ func (h *TraceHandler) runTest(service string, request *traceRequest, tFunc test
 	if err != nil {
 		return err
 	}
-	if err := vFunc(request, traces); err != nil {
-		return err
-	}
-	return nil
+	return vFunc(request, traces)
```
Comment (vprithvi) [inline crossdock/services/tracehandler.go]: These seem to be changes unrelated to this PR.

Comment (yurishkuro) [reply crossdock/services/tracehandler.go]: Because go vet complained

File: .travis.yml [hunk 2]
```diff
@@ -1,6 +1,7 @@
 sudo: required
 
 language: go
+go_import_path: github.com/uber/jaeger
```
Comment (black-adder) [inline .travis.yml]: just so I'm on the same boat, why is this required?

Comment (yurishkuro) [reply .travis.yml]: after the repo move the github path is going to be different, but all the imports in the code refer to uber/jaeger, so it won't  build. This instruction tells Travis to relocate the repo to this path under GOPATH before building.
